src/custom-elements/src/components/tabs/tabs.tsx

```diff
@@ -352,11 +350,32 @@ export class Tabs {
     this.isSeeMoreActive = false;
   }
 
-  isActiveTabOverflown() {
-    return (
-      this.tabs.findIndex(tab => tab.id === this.activeTab && tab.overflow) !==
-      -1
-    );
+  _isActiveTabChildren(tab: TabTrigger) {
+    if (tab.id === this.activeTab) {
+      return true;
+    } else if (tab.children) {
+      return this.isActiveTabOverflown(tab.children, true);
+    }
+  }
```

Comment:
What about unifying these methods into this one?

```suggestion
  _isActiveTab(tab: TabTrigger, isParent: boolean) {
    if (tab.id === this.activeTab && (!isParent || tab.overflow)) {
      return true;
    } else if (tab.children) {
      return this.isActiveTabOverflown(tab.children) && (!isParent || tab.overflow);
    }
  }
```